insights/specs/default.py

```diff
@@ -516,6 +516,10 @@ class DefaultSpecs(Specs):
     resolv_conf = simple_file("/etc/resolv.conf")
     rhsm_conf = simple_file("/etc/rhsm/rhsm.conf")
     rhsm_releasever = simple_file('/var/lib/rhsm/cache/releasever.json')
+    rhui_releasever = first_of([
+        simple_file('/etc/yum/vars/releasever'),
+        simple_file('/etc/dnf/vars/releasever'),
+    ])
```

Comment:
Better change to use `first_file` instead of the `first_of`, and take the `dnf` one as the first item.

```python
    rhui_releasever = first_file(['/etc/yum/vars/releasever',      '/etc/dnf/vars/releasever',])
```